convert2rhel/unit_tests/toolopts_test.py

```diff
@@ -68,3 +71,17 @@ def test_cmdline_disablerepo_defaults_to_asterisk(self):
     @unit_tests.mock(sys, "argv", _params(["--disable-submgr"]))
     def test_cmdline_exits_on_empty_enablerepo_with_disable_submgr(self):
         self.assertRaises(SystemExit, convert2rhel.toolopts.CLI)
+
+
+@pytest.mark.parametrize(
+    ("argv"), [
+        _params(["-v", "Server"]),
+        _params(["--variant", "Client"]),
+        _params(["--variant"])
+    ]
+)
+def test_cmdline_obsolete_variant_option(argv, monkeypatch, caplog):
+    monkeypatch.setattr(sys, "argv", argv)
+    monkeypatch.setattr(convert2rhel.utils, "ask_to_continue", lambda: None)
```

Comment:
lambda: None is fine, but better to create this mock as mock.Mock() object. This will give you a wider set of things you can check. So you can assert if this object was called, check with which arguments it was called, how many times it was called, or that it wasn't called.

Reply:
I've seen that in your other tests. But the question to continue does not seem that important from my POV so I intentionally skipped checking it. But since you're mentioning it, I'll add it :)

Reply:
yes, reviewing helps to see the same things from the different perspective:)